pineappl_py/docs/.gitignore

```diff
@@ -1,9 +0,0 @@
-# autogenerated file
-source/development/code_todos.rst
-# Ignore auto generated module references
-source/modules
```

Comment:
I'd suggest adding this back, eventually

Reply:
Not the last line though, at least for now we manually created the file `pineappl_py/docs/source/modules/pineappl.rst`.

Reply:
done in [09460b6](https://github.com/NNPDF/pineappl/pull/302/commits/09460b6ec1f7502e1c72e75e0ffc1f4cd34b9af4)